react/ProductSummaryTypes.ts

```diff
@@ -12,22 +12,23 @@ export interface State {
 }
 
 export interface Product {
-  linkText: string
-  description: string
-  productName: string
-  brand: string
-  brandId: number
-  categoryId: string
-  categoryTree: Category[]
-  productId: string
-  titleTag: string
-  metaTagDescription: string
-  items: SKU[]
-  skuSpecifications: SkuSpecification[]
-  selectedProperties: SelectedProperty[]
-  productClusters: Array<{ name: string }>
-  priceRange: PriceRangeOptions
-  sku: SingleSKU
+  brand: string;
+  brandId: number;
+  categoryId: string;
+  categoryTree: Category[];
+  description: string;
+  items: SKU[];
+  linkText: string;
+  metaTagDescription: string;
+  priceRange: PriceRangeOptions;
+  productName: string;
+  productId: string;
+  productReference: string;
```

Comment:
It may be a breaking change. I suggest you not enforce this property exists.
```suggestion
  productReference?: string;
```

Reply:
it's good! fixed

Reply:
I was looking that this data already exists in our context, so maybe we can require it on the product type. @victorhmp and @icazevedo can you help us?

Reply:
it is not the only one missing, for example in type of products from [Search Resovler](https://github.com/vtex-apps/search-resolver/blob/master/node/typings/Catalog.ts) has this additional properties
```javascript
{
 "productTitle": string
 "clusterHighlights": Record<string, string>
 "searchableClusters": Record<string, string>
 "categories": string[]
 "categoriesIds": string[]
 "link": string
 "Specifications"?: string[]
 "allSpecifications"?: string[]
 "allSpecificationsGroups"?: string[]
 "completeSpecifications"?: CompleteSpecification[]
}
```